concordium-consensus/src/Concordium/GlobalState/Persistent/BlobStore/Flush.hs

```diff
@@ -0,0 +1,47 @@
+{-# LANGUAGE CPP #-}
+
+-- |This module provides functionality for flushing the operating system buffer associated with
+-- a file handle.
+module Concordium.GlobalState.Persistent.BlobStore.Flush (hFlushOS) where
+
+import Control.Concurrent
+import Control.Monad
+import Foreign.C
+import GHC.IO.Exception
```

Comment:
Unless I forget, this import is not needed on linux, and causes a compilation warning.